src/content/docs/logs/forward-logs/heroku-log-forwarding.mdx

```diff
@@ -32,8 +32,8 @@ To configure this type of drain:
 5. Optional: Click <DNT>**Add metadata**</DNT> if you want to add custom metadata to the log drain and fill up the list with the key pairs you need.
 6. Give your application a name. You'll use it to find your data later.
 7. Copy the Heroku CLI command to your clipboard clicking the <DNT>**Copy to clipboard**</DNT> button.
-8. Go to your terminal in the directory where you have the Heroku CLI set up.
-9. Paste the command and replace `{{YOUR_APP_NAME}}` with your Heroku application name.
+8. Go to your terminal in the directory where you have the Heroku CLI set up.If you’re using Windows VM, use the traditional command prompt instead of Git Bash.
```

Comment:
```suggestion
8. Go to your terminal in the directory where you have the Heroku CLI set up. If you’re using Windows VM, use the traditional command prompt instead of Git Bash.
```